modules/configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery.adoc

```diff
@@ -0,0 +1,123 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-vnet-subnet-iro-discovery_{context}"]
+= Setting up a private storage endpoint on Microsoft Azure by configuring the Image Registry Operator to discover VNet and subnet names
```

Comment:
Maybe try to make the title of this procedure more parallel to the other one - start with "Configuring", write out Microsoft (or don't) both places, etc. That is, if they are two different options, instead of users doing both.